src/main/modules/properties-volume/index.ts

```diff
@@ -0,0 +1,17 @@
+import config from 'config';
+import * as propertiesVolume from '@hmcts/properties-volume';
+import { Application } from 'express';
+import { get, set } from 'lodash';
+
+export class PropertiesVolume {
+
+  enableFor(server: Application): void {
+    if (server.locals.ENV !== 'development') {
+      propertiesVolume.addTo(config);
+
+      if (config.has('secrets.rpe.AppInsightsInstrumentationKey')) {
```

Comment:
Can you abstract this to a method like in https://github.com/hmcts/div-petitioner-frontend/blob/master/app/core/setup/setupSecrets.js#L4